client/src/index.js

```diff
@@ -1,12 +1,34 @@
 import React from 'react';
 import ReactDOM from 'react-dom';
+import { BrowserRouter, Route, Routes } from 'react-router-dom'
 import './index.css';
+import Welcome from './components/Welcome/Welcome'
 import App from './App';
+import Topbar from './components/Topbar/Topbar';
 import reportWebVitals from './reportWebVitals';
+import Login from './components/Login/Login'
+import Register from './components/Register/Register'
+import Protected from './components/Auth/Protected'
+import CreateFlashcard from './components/Flashcard/CreateFlashcard'
+import AuthProvider from './components/Auth/AuthProvider'
+
 
 ReactDOM.render(
   <React.StrictMode>
-    <App />
+    <BrowserRouter>
+      <AuthProvider>
+        <Topbar />
+        <Routes>
+          <Route path="/" element={<Welcome />} />
+          {/* Addignment: Add a route for /user to redirect to from Login */}
```

Comment:
Please remove comments that aren't needed.

Reply:
comments removed